public/doc/swagger-2.yaml

```diff
@@ -298,6 +298,23 @@ paths:
           description: Service Offering not found
         '422':
           $ref: '#/responses/InvalidEntity'
+  '/portfolio_items/{portfolio_item_id}':
+    delete:
+      tags:
+        - admins
+      summary: Delete an existing portfolio item
+      operationId: destroyPortfolioItem
+      description: |
+        Deletes the portfolio item id passed in as the param.
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/PortfolioItemID'
+      responses:
+        204:
```

Comment:
@syncrou The 204 status reads
```
he HTTP 204 No Content success status response code indicates that the request has succeeded, but that the client doesn't need to go away from its current page
```
@lgalis Is the UI going to react properly for a 204, it would have to issue a refresh of the portfolio items page